src/prepareAssetsPackage.js

```diff
@@ -30,6 +30,11 @@ function makePackage({ paths, publicFolders }) {
 
     Object.keys(paths).forEach((assetPath) => {
       const resolvePath = paths[assetPath];
+
+      if (!resolvePath) {
+        throw new Error(`Unable to resolve asset path: ${assetPath}`);
+      }
```

Comment:
Although maybe there is more accurate behavior here, like falling back to assetPath or returning early instead? I don't have much context on this part of the code, so it's hard for me to know what the best thing is.